gammapy/irf/psf/map.py

```diff
@@ -6,6 +6,7 @@
 from gammapy.modeling.models import PowerLawSpectralModel
 from gammapy.utils.gauss import Gauss2DPDF
 from gammapy.utils.random import InverseCDFSampler, get_random_state
+from matplotlib.ticker import FormatStrFormatter
```

Comment:
Matplotlib is still an optional dependency, so this has to be a delayed import in the method where it's used. This also leads to the remaining test fail...

Reply:
ah very good point, I missed that. Fixed now!